spec/ralyxa/response_entities/directives/audio/audio_item_spec.rb

```diff
@@ -0,0 +1,23 @@
+require_relative '../../../../spec_helper'
+
+RSpec.describe Ralyxa::ResponseEntities::Directives::Audio::AudioItem do
+  subject(:audio_item) { described_class.new(Ralyxa::ResponseEntities::Directives::Audio::Stream.new('http://localhost', 'token')) }
+
+  describe '#initialize' do
+    it 'sets the expected instance variables' do
+      expect(audio_item.instance_variable_get(:@stream)).to be_a(Ralyxa::ResponseEntities::Directives::Audio::Stream)
+    end
+  end
+
+  describe '#to_h' do
+    it 'generates the expected hash' do
+      expect(audio_item.to_h).to eq({ "stream" => { "url" => "http://localhost", "token" => "token", "offsetInMilliseconds" => 0 } })
+    end
+  end
+
+  describe '#self.as_hash' do
+    it 'generates the expected hash' do
+      expect(described_class.as_hash(Ralyxa::ResponseEntities::Directives::Audio::Stream.new('http://localhost', 'token'))).to eq({ "stream" => { "url" => "http://localhost", "token" => "token", "offsetInMilliseconds" => 0 } })
```

Comment:
Can we split this up a bit?

Reply:
Do you just mean across multiple lines for readability?